From 86c7f194b0f826b6850deaac7ccfea079fc4a46a Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Wed, 21 Jan 2026 14:19:37 +0000
Subject: [PATCH] Fix ast compatibility for Python 3.14+

Replace node.value.s with node.value.value for compatibility with Python 3.14+.

Upstream-Status: Inappropriate [embedded specific]
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3c38d44..f768533 100644
--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@ class VersionFinder(ast.NodeVisitor):
     def visit_Assign(self, node):
         if not self.version:
             if node.targets[0].id == "__version__":
-                self.version = node.value.s
+                self.version = node.value.value
 
 
 def read_version():
